packages/eslint-config-fbjs/README.md

```diff
@@ -16,23 +16,23 @@ There are several exceptions:
 npm install --save-dev \
   eslint-config-fbjs \
   eslint-plugin-babel \
-  eslint-plugin-flowtype \
+  eslint-plugin-ft-flow \
   eslint-plugin-jsx-a11y \
   eslint-plugin-react \
```

Comment:
theoretically, we probably want to add https://www.npmjs.com/package/eslint-plugin-react-hooks to this config (and make sure the right rules are enabled) but that's distinct from the effort you have now. Fits with my version bump outside PRs though :)